analyses/cell-type-analysis/scripts/01_read-data.R

```diff
@@ -0,0 +1,14 @@
+library(SingleCellExperiment)
+
```

Comment:
Be sure to load in the `readr` package first! 
```suggestion
library(readr)
```